meta/system.py

```diff
@@ -535,12 +565,19 @@ def linear_alignment(self, name, corpus, flow, prefix="", **kwargs):
                 },
             )
 
-    def align(self, name, corpus, flow, feature_scorer, **kwargs):
+    def align(
+        self,
+        name: str,
+        corpus: str,
+        flow: Union[str, List[str], Tuple[str], rasr.FlowNetwork],
+        feature_scorer: Union[str, List[str], Tuple[str], rasr.FeatureScorer],
```

Comment:
actually `Tuple[str]` means a Tuple containing a single string. But additionally allowed would be 
`Tuple[str, str]` and `Tuple[str, str, int]`

`List[str]` is also only so-so correct, as only Lists with the elements equivalent to the tuples above would be possible.. Well technically also a list with (str, str, int, Any, Any, ...) would work, as all elements behind list[2] are ignored.. But I don't know a good way to type this..

However in most setups the Tuple version is used in practice I think, so whenever we decide to use typechecking I think the tuples should be typed correctly.

Maybe we should define a `selector_type` to make the union shorter.. i.e.
```
selector_type = Union[str, List, Tuple[str], Tuple[str, str], Tuple[str, str, int]]

[...]

flow : Union[selector_type, rasr.FlowNetwork],
feature_scorer: : Union[selector_type, rasr.FeatureScorer],